packages/create-block/lib/templates/es5/readme.txt.mustache

```diff
@@ -0,0 +1,54 @@
+=== {{title}} ===
+Contributors:      {{author}}
+Tags:              comments, spam
+Requires at least: 5.0
+Tested up to:      5.3.2
+Stable tag:        {{version}}
+Requires PHP:      7.0.0
+License:           {{license}}
+License URI:       License URL
```

Comment:
It feels like we could add another variable to enter `{{licenseURI}} that defaults to https://www.gnu.org/licenses/gpl-2.0.html.